string handConf
---
int64 flag